fuzzing/get-afl.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
 cd "${0%/*}"
-rm afl-latest.tgz
```

Comment:
I think these all make sense, but a quick summary of why you made these changes to `get-afl.sh` would be helpful.